src/main/java/se/iths/librarysystem/controller/BookController.java

```diff
@@ -48,4 +50,18 @@ public BookController(BookService bookService, ModelMapper modelMapper) {
                 .body(savedBook);
     }
 
-}
+    @DeleteMapping("{id}")
+    public ResponseEntity<Void> deleteBook(@PathVariable Long id){
+
+        bookService.deleteBook(id);
+        return new ResponseEntity<>(HttpStatus.NO_CONTENT);
+    }
+
+    @GetMapping("{id}")
+    public ResponseEntity<Book> findBookByID(@PathVariable Long id) {
+        BookEntity foundBook = bookService.findBookById(id).orElseThrow(() -> new IdNotFoundException("books", id));
+        // convert BookEntity to a Book DTO
+        return new ResponseEntity<>(new Book(), HttpStatus.OK);
```

Comment:
```suggestion
        Book book = modelMapper.map(foundBook, Book.class);
        return new ResponseEntity<>(book, HttpStatus.OK);
```

If we return `new Book()` it will be an empty book without any id, title, edition, isbn etc.

Reply:
Done,  you can review again.